docker/pypi/wmagent-couchdb/README.md

```diff
@@ -0,0 +1,324 @@
+21d9c6598f35e627834d1b796460047605d6255cebc746d572289c7b418053ed
+
+cmst1@vocms0290:wmagent-couchdb $ docker ps
+CONTAINER ID   IMAGE                             COMMAND      CREATED         STATUS         PORTS     NAMES
+21d9c6598f35   registry.cern.ch/couchdb:3.2.2   "./run.sh"   7 seconds ago   Up 6 seconds             couchdb
+
+```
+
+* Killing the container directly from the host:
+```
+cmst1@vocms0290:wmagent-couchdb $ docker kill couchdb
+couchdb
+
+```
+
+* Connecting to a running container:
+```
+cmst1@vocms0290:wmagent-couchdb $ docker exec -it couchdb bash
+(CouchDB-3.2.2) [cmst1@vocms0290:data]$
+
+```
+
+* Fetching startup logs:
+```
+cmst1@vocms0265:wmagent-couchdb $ docker logs couchdb
+ME : data
+TOP : /
+ROOT : /
+CFGDIR : /data/srv/couchdb/3.2.2/config
+LOGDIR : /data/srv/couchdb/3.2.2/logs
+STATEDIR : /data/srv/couchdb/3.2.2/state
+KEYFILE : /data/srv/couchdb/auth//hmackey.ini
+
+COUCH_ROOT_DIR : /data
+COUCH_BASE_DIR : /data/srv/couchdb
+COUCH_STATE_DIR : /data/srv/couchdb/3.2.2/state
+COUCH_INSTALL_DIR : /data/srv/couchdb/3.2.2/install
+COUCH_CONFIG_DIR : /data/srv/couchdb/3.2.2/config
+<snip>
+Installing WorkQueue app into database: http://localhost:5984/workqueue
+Installing WorkQueue app into database: http://localhost:5984/workqueue_inbox
+start sleeping....zzz
+```
+
+### Managing the database service:
+
+All of the commands bellow must be run from inside the container.
+
+* General options:
+```
+(CouchDB-3.2.2) [cmst1@vocms0290:data]$ manage help
+
+ME : 3.2.2
+TOP : /data
+ROOT : /data/srv
+CFGDIR : /data/srv/couchdb/3.2.2/config
+LOGDIR : /data/srv/couchdb/3.2.2/logs
+STATEDIR : /data/srv/couchdb/3.2.2/state
+KEYFILE : /data/srv/couchdb/auth//hmackey.ini
+
+COUCH_ROOT_DIR : /data
+COUCH_BASE_DIR : /data/srv/couchdb
+COUCH_STATE_DIR : /data/srv/couchdb/3.2.2/state
+COUCH_INSTALL_DIR : /data/srv/couchdb/3.2.2/install
+COUCH_CONFIG_DIR : /data/srv/couchdb/3.2.2/config
+
+Usage: manage ACTION [ARG] [SECURITY-STRING]
+
+Available actions:
+  help            show this help
+  version         get current version of the service
+  status          show current service's status
+  sysboot         start server from crond if not running
+  restart         (re)start the service
+  start           (re)start the service
+  stop            stop the service
+  pushapps        push couch applications
+  pushreps        push couch replications
+  updatecouchapps pull new couch applications from WMCore repo
+  compact         compact database ARG
+  compactviews    compact database views for design doc ARG ARG
+  cleanviews      clean view named ARG
+  backup          rsync databases to ARG (i.e. [user@]host:path)
+  archive         archive backups to ARG area in castor
+
+For more details please refer to operations page:
+  https://twiki.cern.ch/twiki/bin/view/CMS/CouchDB
+
+```
+
+* Start/Stop the database server:
+```
+(CouchDB-3.2.2) [cmst1@vocms0290:data]$ manage start
+ME : 3.2.2
```

Comment:
I am not sure how much help is printing all these variables for every `manage` command executed. How about making a new manage option like `print-env` or something like this that would print those only when we ask for it?

Reply:
done

Reply:
I also think we can give an example of `print-settings` output and remove all the repetition of these variables in this README.